src/bramble-live-dev-cache-sw.js

```diff
@@ -5,6 +5,24 @@
  * which is named vfs/project/root.
  */
 
+// String comes from src/filesystem/impls/filer/UrlCache.js
+var liveDevUrlRegex = /thimble-sw-vfs-cached-url\//;
+
+function custom500() {
+    "use strict";
+
+    var body = "<!doctype html><title></title><p>There was an error serving your content. Try restarting your web browser to clear your cache.";
```

Comment:
yeah we might need to somehow do a static build. How do we tell the client to request the appropriate localized file though?